.github/dockerized-gh_actions_runners/Kubernetes/github-actions-windows-deployment.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: iis
+  labels:
+    app: iis
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: iis
+  template:
+    metadata:
+      labels:
+        app: iis
+    spec:
+      nodeSelector:
+        kubernetes.io/os: windows
+      containers:
+      - name: iis-server
+        image:  #todo: replace with gcr, for now im using docker hub img
+        env:
+        - name: GITHUB_REPO
+          value: "Modify with your own repo"
+        - name: GITHUB_TOKEN
+          value: "Modify with your token"
```

Comment:
As the token is a sensitive variable, we need to figure out how to use an encrypted variable. Thank you

Reply:
Sure, I implemented something, will talk about it on this evening meeting.